src/supertux/game_session.hpp

```diff
@@ -135,12 +141,17 @@ class GameSession final : public Screen,
 
   void on_escape_press(bool force_quick_respawn);
 
+  Vector get_fade_point() const;
+  Vector get_fade_point(const Vector& position) const;
+
 public:
   bool reset_button;
   bool reset_checkpoint_button;
 
   bool m_prevent_death; /**< true if players should enter ghost mode instead of dying */
 
+  static constexpr float TELEPORT_FADE_TIME = 1.0f; /**< Duration of teleport fade animation */
+
```

Comment:
Instead, use a static variable, located at the start of the class declaration, prefixed with `s_`:

```
public:
  static const float s_teleport_fade_time;
```

The static variable should be defined at the top of the .cpp file:

```
const float GameSession::s_teleport_fade_time = 1.f;
```

Reply:
Sorry, somehow I missed this. It could be moved completely to the implementation file as the GameSession will make Tux safe during the fadeout animation anyway. I hope it is Ok that I did it similar to the other static const's in the other code instead of using your suggestion. Otherwise I can change it again.

Reply:
It's okay, my suggestion was keeping in mind usage from other files.